docs/platform/concepts/permissions.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Roles and permissions
+---
+
+To give users access to projects and services in your organizations, you grant them permissions and roles:
+
+* **Permissions**: Actions that a principal can perform on a resource or
+  group of resources.
+* **Roles**: Sets of permissions that you can assign to a principal.
+
+Principals are
+[organization users](/docs/platform/howto/manage-org-users),
+[application users](/docs/platform/concepts/application-users),
+and [groups](/docs/platform/howto/list-groups).
+You can grant access to principals at the project level.
+
+To grant access to resources at the organization level, you can
+make organization users [super admin](/docs/platform/howto/make-super-admin).
```

Comment:
```suggestion
assign [super admin](/docs/platform/howto/make-super-admin) role to organization users.
```
"make" can be ambiguous.

Reply:
It's the language we use in the UI and the doc that is linked `/docs/platform/howto/make-super-admin`.